packages/dd-trace/src/appsec/iast/analyzers/ssrf-analyzer.js

```diff
@@ -0,0 +1,25 @@
+'use strict'
+
+const InjectionAnalyzer = require('./injection-analyzer')
+
+class SSRFAnalyzer extends InjectionAnalyzer {
+  constructor () {
+    super('SSRF')
```

Comment:
Should use the enum from [packages/dd-trace/src/appsec/iast/vulnerabilities.js](https://github.com/DataDog/dd-trace-js/pull/3115/files#diff-359177f30c6a634464ec2d384a56ffb26eaacd102dabed0ba430ff1e7571d0d9)